code/datums/station_traits/job_traits.dm

```diff
@@ -174,6 +176,73 @@
 	. = ..()
 	overlays += "veteran_advisor"
 
+/datum/station_trait/job/human_ai
+	name = "Human AI"
+	button_desc = "Sign up to become the \"AI\"."
+	weight = 1
+	trait_flags = parent_type::trait_flags | STATION_TRAIT_REQUIRES_AI
+	report_message = "Our recent technological advancements in machine Artificial Intelligence has proven futile. In the meantime, we're sending an Intern to help out."
+	show_in_report = TRUE
+	can_roll_antag = CAN_ROLL_PROTECTED
+	job_to_add = /datum/job/human_ai
+	trait_to_give = STATION_TRAIT_HUMAN_AI
+
+/datum/station_trait/job/human_ai/New()
+	. = ..()
+	RegisterSignal(SSjob, COMSIG_OCCUPATIONS_SETUP, PROC_REF(remove_ai_job))
+	RegisterSignal(SSatoms, COMSIG_SUBSYSTEM_POST_INITIALIZE, PROC_REF(give_fax_machine))
+
+/datum/station_trait/job/human_ai/revert()
+	UnregisterSignal(SSjob, COMSIG_OCCUPATIONS_SETUP)
+	UnregisterSignal(SSatoms, COMSIG_SUBSYSTEM_POST_INITIALIZE)
+	return ..()
+
+/datum/station_trait/job/human_ai/on_lobby_button_update_overlays(atom/movable/screen/lobby/button/sign_up/lobby_button, list/overlays)
+	. = ..()
+	overlays += LAZYFIND(lobby_candidates, lobby_button.get_mob()) ? "human_ai_on" : "human_ai_off"
+
+/datum/station_trait/job/human_ai/proc/remove_ai_job(datum/source)
+	SIGNAL_HANDLER
+	var/datum/job_department/department = SSjob.joinable_departments_by_type[/datum/job_department/silicon]
+	department.remove_job(/datum/job/ai)
+	var/datum/station_trait/triple_ai/triple_ais = locate() in SSstation.station_traits
+	if(triple_ais)
+		position_amount = 3
+
+/// Gives the AI SAT a fax machine if it doesn't have one. This is copy pasted from Bridge Assistant's coffee maker.
+/datum/station_trait/job/human_ai/proc/give_fax_machine(datum/source)
+	SIGNAL_HANDLER
+	var/area/sat_area = GLOB.areas_by_type[/area/station/ai_monitored/turret_protected/ai]
+	if(isnull(sat_area))
+		return
+	var/list/fax_machines = SSmachines.get_machines_by_type_and_subtypes(/obj/machinery/fax)
+	for(var/obj/machinery/fax_machine as anything in fax_machines) //don't spawn a coffeemaker if there is already one on the bridge
+		if(is_type_in_list(get_area(fax_machine), sat_area))
+			return
+	var/list/tables = list()
+	for(var/turf/area_turf as anything in sat_area.get_turfs_from_all_zlevels())
+		var/obj/structure/table/table = locate() in area_turf
+		if(isnull(table))
+			continue
+		if(area_turf.is_blocked_turf(ignore_atoms = list(table)))
+			continue
+		tables += table
+	if(!length(tables))
+		return
+	var/picked_table = pick_n_take(tables)
```

Comment:
I don't quite like how, instead of looping through all possible tables to find a good one, it throws two shots in the dark.

Reply:
All tables are 'good' ones, it just moves things to another table (if more than 1 table exists in the area).

Reply:
Oh my bad.